test/Microsoft.SqlTools.ServiceLayer.Test.Common/TestConnectionProfileService.cs

```diff
@@ -0,0 +1,206 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.SqlTools.ServiceLayer.Connection.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Credentials;
+using Microsoft.SqlTools.ServiceLayer.Credentials.Contracts;
+using Microsoft.SqlTools.ServiceLayer.TestDriver.Driver;
+
+namespace Microsoft.SqlTools.ServiceLayer.Test.Common
+{
+    /// <summary>
+    /// Service to get connection profiles from the configured settings
+    /// to get the credentials, test driver will be used if available otherwise the credential service will be called directly
+    /// </summary>
+    public class TestConnectionProfileService
+    {
+        public static IEnumerable<TestServerIdentity> TestServers = InitTestServerNames();
+        public static ConnectionSetting Setting = InitSetting();
```

Comment:
Shouldn't these be Lazy? Otherwise any Assembly reference will cause this code to be run. Also please make this a property, or at least mark as Readonly.